diff --git a/src/base64.c b/src/base64.c
index f254e5a..bf44fc3 100644
--- a/src/base64.c
+++ b/src/base64.c
@@ -7,7 +7,7 @@
  */
 
 #include <string.h>
-#include "../include/orcania.h"
+#include "orcania/orcania.h"
 
 static const unsigned char base64_table[65] =
 	"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
diff --git a/src/memory.c b/src/memory.c
index a3415dd..3d502e7 100644
--- a/src/memory.c
+++ b/src/memory.c
@@ -9,7 +9,7 @@
  */
 
 #include <stdlib.h>
-#include "../include/orcania.h"
+#include "orcania/orcania.h"
 
 /* memory function pointers */
 static o_malloc_t do_malloc = malloc;
diff --git a/src/orcania.c b/src/orcania.c
index 219cd74..707f57b 100644
--- a/src/orcania.c
+++ b/src/orcania.c
@@ -31,7 +31,7 @@
 #include <string.h>
 #include <ctype.h>
 #include <stdarg.h>
-#include "../include/orcania.h"
+#include "orcania/orcania.h"
 
 #ifdef _MSC_VER
 #define strncasecmp _strnicmp
